dockstore-client/src/test/java/io/dockstore/client/cli/LaunchTestIT.java

```diff
@@ -716,4 +716,40 @@ public void cwlNoInput() throws IOException {
 
     }
 
+    @Test
+    public void toolAsWorkflow() {
+        File cwlFile = new File(ResourceHelpers.resourceFilePath("dir6.cwl"));
+        File cwlJSON = new File(ResourceHelpers.resourceFilePath("dir6.cwl.json"));
+
+        ArrayList<String> args = new ArrayList<String>() {{
+            add("workflow");
+            add("launch");
+            add("--local-entry");
+            add(cwlFile.getAbsolutePath());
+            add("--json");
+            add(cwlJSON.getAbsolutePath());
+        }};
+        exit.expectSystemExit();
+        runClientCommand(args, false);
+        assertTrue("Out should suggest to run as tool instead", systemOutRule.getLog().contains("Expected a workflow but the"));
```

Comment:
FYI, @k-cao   discovered that using assertTrue in this way won't work, use "checkAssertionAfterwards" instead 
https://stefanbirkner.github.io/system-rules/